### PR TITLE
Implemented option to run jobs in parallel for PV validation

### PR DESCRIPTION
**Pull request by cardinia**
@connorpa 
Option for running jobs in parallel for the Primary Vertex Validation implemented and tested as discussed in Tracker Alignment meeting (https://indico.cern.ch/event/750047/#35-parallelisation-of-validati)

### CODE REVIEW

**Comment by tocheng** [comment]
@cmsbuild please test

File: Alignment/OfflineValidation/python/TkAlAllInOneTool/primaryVertexValidation.py [hunk 1]
```diff
@@ -28,9 +28,9 @@ def __init__(self, valName, alignment, config):
 
         if self.general["pvvalidationreference"].startswith("/store"):
             self.general["pvvalidationreference"] = "root://eoscms//eos/cms" + self.general["pvvalidationreference"]
-        if self.NJobs > 1:
-            raise AllInOneError("Parallel jobs not implemented for the PrimaryVertex validation!\n"
-                                "Please set parallelJobs = 1.")
+#        if self.NJobs > 1:
```
Comment (mmusich) [inline Alignment/OfflineValidation/python/TkAlAllInOneTool/primaryVertexValidation.py]: @cardinia why keep the commented out lines?

Comment (cardinia) [reply Alignment/OfflineValidation/python/TkAlAllInOneTool/primaryVertexValidation.py]: @mmusich , they can be removed without problems. I initially thought that keeping them could be useful when working on the Parallel Validation code itself to loosen some of the current limitations (number of jobs to be sent in parallel or the need to specify the maximum number of events). But that was just a thought, their usefulness is basically none, I can remove those lines without problems.

**Comment by fabiocos** [comment]
@smuzaffar not sure why the status of test and comparison was not updated

@lpernie @tocheng could you please review this PR?

**Comment by tocheng** [comment]
+1

**Comment by smuzaffar** [comment]
@fabiocos , PR was updated that is why test and comparison signatures were sent to pending.

**Comment by fabiocos** [comment]
+1

**Pull request merged**